databuilder/job/job.py

```diff
@@ -48,7 +48,8 @@ def init(self, conf):
 
     def _init(self):
         # type: () -> None
-        self.task.init(self.conf)
+        task_conf = Scoped.get_scoped_conf(self.conf, self.task.get_scope())
+        self.task.init(task_conf.with_fallback(self.conf))
```

Comment:
didn't know we have this option before :)

Reply:
Yep, hocon config is quite powerful :)